tests/test_model_object.py

```diff
@@ -0,0 +1,25 @@
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+
+from keras_generators.model_abstractions.model_object import ModelObject
+
+
+class TestModelObject(TestCase):
+    def setUp(self):
+        mp = MagicMock()
+        model = MagicMock()
+        encoders = MagicMock()
+        self.model_object = ModelObject(mp, model, encoders)
+
+    @patch('keras.backend.clear_session')
+    def test_check_clear_state(self, mock_clear_session):
```

Comment:
All is good, with the exception that there's no point to test a private method. We need to test the public methods that call the backend.clear_session after a specified number of calls, where number of calls depends on the state_autoclear argument

Reply:
Done